kolibri/plugins/learn/assets/src/my_downloads/views/MyDownloads.vue

```diff
@@ -95,6 +95,7 @@
       const downloads = ref({});
```

Comment:
So... I imagine maybe this was done because of the lack of separation I highlighted above, but this should all be handled in the composable and not need to define additional reactive values here.